frontend/ui/data/onetime_migration.lua

```diff
@@ -697,5 +697,16 @@ if last_migration_date < 20240731 then
     end
 end
 
+-- 20240911, Defaults: Deprecate DKOPTREADER_CONFIG_DOC_LANGS_TEXT after #11977, https://github.com/koreader/koreader/pull/12504
+if last_migration_date < 20240911 then
```

Comment:
Still want to keep the migration with the improved behavior?

Reply:
Yeah, it's fairly trivial, showcases the use of `hasBeenCustomized` vs. `has`, and makes for cleaner files ;).